src/crypto.rs

```diff
@@ -2,6 +2,8 @@ use super::model::Entity;
 use crate::{model::TripleMask, rdf_types::*};
 use rand::Rng;
 
+const MIN_SECRET_SIZE: usize = 32;
```

Comment:
I think this should be defined at the algorithm level, not globally. Some algorithms may require different min secret sizes. For example AES can use 128bits keys.